### PR TITLE
Feature/improve remediation nov2020

### PR DESCRIPTION
**Pull request by grabnerandi**
WIP

### CODE REVIEW

**Comment by grabnerandi** [comment]
This PR implements a couple of new capabilities as described in the readme
1: Additional mapping options of Dynatrace Problems to Keptn project, service & stage
2: Sends auto-remediation action events (remediation started, action started, action finished and evaluation-done) to the Dynatrace PRoblem as comment

File: pkg/common/common.go [hunk 1]
```diff
@@ -8,10 +8,19 @@ import (
...
+
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
+
+	keptnapi "github.com/keptn/go-utils/pkg/api/utils"
+	keptn "github.com/keptn/go-utils/pkg/lib"
 )
 
+// This is the label name for the Problem URL label
+var PROBLEMURL_LABEL = "Problem URL"
```
Comment (bacherfl) [inline pkg/common/common.go:21]: this can be changed to `const` instead of `var`

File: pkg/common/common.go [hunk 2]
```diff
@@ -8,10 +8,19 @@ import (
...
+var KEPTNSBRIDGE_LABEL = "Keptns Bridge"
```
Comment (bacherfl) [inline pkg/common/common.go:22]: this can be changed to `const` instead of `var`

**Review by bacherfl — CHANGES_REQUESTED** [review]
Looks good! Just added two minor comments. When they have been addressed we can merge the PR

**Review by bacherfl — APPROVED** [review]

**Pull request merged**